wolfcrypt/src/evp.c

```diff
@@ -656,6 +673,69 @@ static int wolfSSL_EVP_CipherUpdate_GCM(WOLFSSL_EVP_CIPHER_CTX *ctx,
 }
 #endif /* HAVE_AESGCM */
 
+#if defined(HAVE_AESCCM)
+#ifndef WOLFSSL_AESCCM_STREAM
```

Comment:
Please add `WOLFSSL_AESCCM_STREAM` to configure like AES GCM is. It should be turned on when the compatibility layer is enabled and CCM is enabled.

Reply:
We do not have full support for streaming in CCM, so it is bypassed and there are code blocks that specifically return an error code when AESCCM_STREAM is attempted.